lib/ansible/modules/slurp.py

```diff
@@ -99,20 +98,18 @@ def main():
 
     try:
         with open(source, 'rb') as source_fh:
-            source_content = source_fh.read()
-    except (IOError, OSError) as e:
-        if e.errno == errno.ENOENT:
-            msg = "file not found: %s" % source
-        elif e.errno == errno.EACCES:
-            msg = "file is not readable: %s" % source
-        elif e.errno == errno.EISDIR:
-            msg = "source is a directory and must be a file: %s" % source
+            data = base64.b64encode(source_fh.read())
+    except OSError as ex:
+        if ex.errno == errno.ENOENT:
+            msg = f"file not found: {source!r}"
+        elif ex.errno == errno.EACCES:
+            msg = f"file is not readable: {source!r}"
+        elif ex.errno == errno.EISDIR:
+            msg = f"source is a directory and must be a file: {source!r}"
         else:
-            msg = "unable to slurp file: %s" % to_native(e, errors='surrogate_then_replace')
+            msg = "unable to slurp file"
```

Comment:
This fits in with the suggestion above:

```suggestion
            msg = "Unable to slurp file: {source}"
```